### PR TITLE
Add aarch64 wheel build support

### PR DESCRIPTION
**Pull request by odidev**
Added linux aarch64 wheel support.

Related to: https://github.com/OpenNMT/Tokenizer/issues/273

### CODE REVIEW

**Comment by odidev** [comment]
@guillaumekln 
Yes, I have checked the same, In GitHub Actions the build for AArch64 took an hour and 15 minutes on average.
please check this link for more details: https://github.com/odidev/Tokenizer/actions/runs/1471078391

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -76,18 +76,26 @@ jobs:
...
 
   build-and-test-python-wheels:
-    name: Build and test Python wheels on ${{ matrix.os }}
+    name: Build and test wheels on ${{matrix.arch}} for ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
         os: [ubuntu-20.04, macos-10.15, windows-2019]
+        arch: [auto]
```
Comment (guillaumekln) [inline .github/workflows/ci.yml:85]:
```suggestion
        arch: [auto64]
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -102,6 +110,7 @@ jobs:
           CIBW_TEST_COMMAND: pytest {project}/bindings/python/test/test.py
           CIBW_TEST_REQUIRES: pytest
           CIBW_ARCHS: auto64
+          CIBW_ARCHS_LINUX: ${{matrix.arch}}
```
Comment (guillaumekln) [inline .github/workflows/ci.yml:113]:
```suggestion
          CIBW_ARCHS: ${{matrix.arch}}
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -76,18 +76,26 @@ jobs:
 
 
   build-and-test-python-wheels:
-    name: Build and test Python wheels on ${{ matrix.os }}
+    name: Build and test wheels on ${{matrix.arch}} for ${{ matrix.os }}
```
Comment (guillaumekln) [inline .github/workflows/ci.yml:79]: I think the other way around is more correct?

```suggestion
    name: Build and test wheels on ${{ matrix.os }} for ${{matrix.arch}} 
```

**Review by guillaumekln — CHANGES_REQUESTED** [review]

**Review by guillaumekln — APPROVED** [review]

**Pull request merged**